### PR TITLE
MGDOBR-967: Add action definition test

### PR DESCRIPTION
**Pull request by sjamboro**
https://issues.redhat.com/browse/MGDOBR-967

Create new test suite for the configuration of Processor's actions.

### CODE REVIEW

File: src/app/Processor/ProcessorEdit/ConfigurationForm/ConfigurationForm.tsx [hunk 1]
```diff
@@ -62,7 +62,13 @@ const ConfigurationForm = (props: ConfigurationFormProps): JSX.Element => {
       disabled={readOnly}
     >
       {Object.keys(properties as { [key: string]: unknown }).map((key) => (
-        <AutoField key={key} name={key} disabled={readOnly} />
+        <AutoField
+          key={key}
+          name={key}
+          disabled={readOnly}
+          data-ouia-component-id={key}
+          data-ouia-component-type="SE/parameter"
```
Comment (jomarko) [inline src/app/Processor/ProcessorEdit/ConfigurationForm/ConfigurationForm.tsx:70]: I am little bit unsure about the chosen type name. Please correct me if I am wrong. I think `SE` stands for `Smart Events`. 

As the changes are done in `ProcessorEdit` folder, I am not sure if the component is common for whole `Smart Events` app, or just for `Processors`.

If  we keep your proposal, my expectations would be `Bridge parameters` and `Processor parameters` are visually totally the same and alsi corresponding html has the same naming conventions, is that the case?

File: src/app/Processor/ProcessorEdit/ProcessorEdit.tsx [hunk 2]
```diff
@@ -431,7 +431,11 @@ const ProcessorEdit = (props: ProcessorEditProps): JSX.Element => {
                                 </p>
                               )}
                             </FormSection>
-                            <FormSection title={t("processor.action")}>
+                            <FormSection
+                              title={t("processor.action")}
+                              data-ouia-component-id="actions"
+                              data-ouia-component-type="SE/FormSection"
```
Comment (jomarko) [inline src/app/Processor/ProcessorEdit/ProcessorEdit.tsx:437]: similar question as above

**Review by jomarko — COMMENTED** [review]
Thank you for a PR @sjamboro

File: src/app/Processor/ProcessorEdit/ConfigurationForm/ConfigurationForm.tsx (hunk 1 above)
Comment (sjamboro) [reply src/app/Processor/ProcessorEdit/ConfigurationForm/ConfigurationForm.tsx:70]: I am trying to use the namespace definition similar as "PF4" which is the name of the whole project not a part of them. I understand that `parameter` might be a poor naming. @jomarko what do you thing about `SE/config-parameter` or `SE/processor-parameter` I am not sure if I am able to know if it is `action` or `source` parameter at this level of code.

File: src/app/Processor/ProcessorEdit/ProcessorEdit.tsx (hunk 2 above)
Comment (sjamboro) [reply src/app/Processor/ProcessorEdit/ProcessorEdit.tsx:437]: @jomarko same situation it is named by the RHOSE project -> SE as SmartEvent project

**Comment by sjamboro** [comment]
@jomarko I have checked that the component type is using by developers too, so I have decited to follow the current format.

**Review by jomarko — APPROVED** [review]

**Review by kelvah — APPROVED** [review]

**Review by ChughShilpa — APPROVED** [review]
LGTM. Thanku for the PR @sjamboro

**Pull request merged**